core/src/main/scala/org/apache/spark/serializer/SerializerManager.scala

```diff
@@ -19,17 +19,22 @@ package org.apache.spark.serializer
 
 import java.io.{BufferedInputStream, BufferedOutputStream, InputStream, OutputStream}
 import java.nio.ByteBuffer
+import javax.crypto.KeyGenerator
 
+import org.apache.hadoop.security.Credentials
```

Comment:
hmm, not sure how this passed the style checker... but it should be in a separate group after `scala.*`.